static/js/addons-config/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="theme-color" content="#75AACF">
+    <title>JSON schema sample</title>
+    <link rel="icon" href="../../images/icon.png" type="image/png" />
+
+    <link rel="stylesheet" type="text/css" href="../../css/app.css" />
+
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/ajv/6.2.1/ajv.min.js"></script>
```

Comment:
I would prefer to pull AJV into `static/js/lib/` such that the UI can continue to work without an internet connection.